packages/shopify-codemod/test/fixtures/remove-empty-returns/arrow-functions.input.js

```diff
@@ -0,0 +1,4 @@
+$document.on('click', 'a.disabled, a.btn-disabled', event => {
+  event.preventDefault();
+  return;
+});
```

Comment:
I still don't see the sanity checks for nested blocks and for returns with arguments:

```js
function foo() {
  if (bar) { return; }
  doSomething();
  return;
}
```

```js
function foo() {
  return 'bar';
}
```